tox.ini

```diff
@@ -28,7 +28,9 @@ setenv =
 commands_pre =
     piplatest: python -m pip install -U pip
     pip --version
-commands = pytest {posargs}
+commands = 
+    py38: pytest --fail-under=98 {posargs}
+    !py38: pytest {posargs}
```

Comment:
Wonder if this doesn't work due to `TOXENV=pipsupported-coverage` rather than `py38`. Hopefully, using those pragmas (https://github.com/jazzband/pip-tools/pull/2253/files#r2501228610) will help.